h/services/user.py

```diff
@@ -49,6 +49,7 @@ def fetch(self, userid_or_username, authority=None):
           user_service.fetch('foo', 'example.com')
 
         :returns: a user instance, if found
+        :raises InvalidUserId: If the username cannot be parsed
```

Comment:
To avoid confusion, we might want to be more precise here that it's the `userid` that can't be parsed (`userid` is composed of `<username>@<authority>`).

Reply:
So what you're saying is that words have meanings, and you can't use them randomly and interchangeably? Controversial... but ok.

Reply:
Best comment ever.